tests/kernel/dsp_sharing/generic/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Synopsys
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(dsp_sharing)
+
+zephyr_include_directories(
+      ${ARCMWDT_TOOLCHAIN_PATH}/MetaWare/arc/lib/src/fx/include/
```

Comment:
this should have some condition to only include for ARC MWDT, or the test should be made ARC specific.

Reply:
fixed